packages/webapp/pages/onboarding.tsx

```diff
@@ -290,21 +270,32 @@ export function OnboardPage(): ReactElement {
     );
   };
 
+  const [isPreviewVisible, setPreviewVisible] = useState(false);
+
   const getContentV3 = (): ReactElement => {
+    const tagsCount = feedSettings?.includeTags?.length || 0;
+    const isPreviewEnabled = tagsCount >= requiredTagsThreshold;
+
     if (isAuthenticating && !isFiltering) {
       return getAuthOptions();
     }
 
     return (
       <div
         className={classNames(
-          'flex tablet:flex-1 laptop:max-w-[37.5rem]',
-          isFiltering
-            ? 'flex-col items-center ml-0 tablet:max-w-[32rem] laptop:max-w-[48.75rem]'
-            : 'ml-auto',
+          'flex tablet:flex-1',
+          !(isFiltering && onboardingV4 === OnboardingV4.V4) &&
```

Comment:
This whole block is called `getContentV3`.
I think initially the idea was we have `getContentV4` for instance instead of all these exclusions.

If you do want to keep them. please rename to `getContent` or simply move it back inline again.

Reply:
Yes, I am cleaning up v2 and v3 still.

Reply:
Cleaned in a75e309f2abb7f67d22d3f30fcca91a551a6de9c, v2 is still left on feed to preserve days left notice.